images/spz/spz/templates/internal/registrations.html

```diff
@@ -22,4 +22,14 @@ <h3 class="ui dividing header">Import</h3>
         <button type="submit" class="fluid ui positive button">Einlesen</button>
     </form>
 </div>
+
+<div class="row">
+    <h3 class="ui dividing header">Matrikelnummer/Kürzel-Überprüfung</h3>
+    <form class="ui form" method="post" action="{{ url_for('registrations_verify') }}" enctype="multipart/form-data">
+        {{ csrf_field() }}
+        {{ render_input(form.tag) }}
+        {{ render_submit(submit='Überprüfen') }}
+
+    </form>
```

Comment:
Instead of a flash message, the result of the search/request should be appear here.